.github/CHANGELOG.md

```diff
@@ -34,6 +34,9 @@
 
 ### Improvements
 
+* Merge `lightning.gpu` and `lightning.tensor` GPU tests in single Python and C++ CIs controlled by the `ci:use-gpu-runner` label.
+  [(#911)](https://github.com/PennyLaneAI/pennylane-lightning/pull/911)
+
```

Comment:
How is it supposed to improve our CIs as of now every change in either LGPU or LT will trigger both LGPU and LT test suites? :thinking:

Reply:
This is a good point and a consequence of merging the workflows as requested.
I will need to think about that.

Reply:
Maybe we need to explore a different strategy, without using a strategy/matrix. Let me think,

Reply:
Wait. Isn't it your intention to always run the tests if we set the label? In this case, both would run, right?
I will need you to clarify that part.

Reply:
We talked about merging the C++ and Python actions for each device; nothing about merging LGPU and LT. But this is a good idea! No blocker from my side if checking Python unit tests of both LGPU and LT can be done in less than 10min in total. We can remove the `path` and `path-ignore` references to ensure that these two actions will run with this GH label. 

Is this difficult to merge C++ and Python actions?

Reply:
As we talked offline, this seems to be a reasonable approach and it unblocks @multiphaseCFD and @LuisAlfredoNu  for their work on LGPU and LT, but let's remove the `path` and `path-ignore` references and only rely on the label for now. Thanks :)